srcpkgs/renderdoc/template

```diff
@@ -0,0 +1,25 @@
+# Template file for 'renderdoc'
+pkgname=renderdoc
+version=1.1
+revision=1
+build_style=cmake
+configure_args="-DENABLE_GL=ON -DENABLE_GLES=ON -DENABLE_PYRENDERDOC=ON
+-DENABLE_QRENDERDOC=ON -DENABLE=RENDERDOCCMD=ON -DENABLE_VULKAN=ON
+-DENABLE_XCB=ON -DENABLE_XLIB=ON"
+hostmakedepends="bison autoconf automake  pkg-config python3-devel"
```

Comment:
I think it should be `python3` in `hostmakedepends` and `pyton3-devel` in `makedepends`. Currently cross-compilation fails because it cannot find the python headers which are only installed for the host.